README.md

```diff
@@ -1,85 +1,64 @@
-  Template uses setuptools because it's the de-facto standard for Python packages, you can run `make switch-to-poetry` later if you want.
-- 🤖 A [Makefile](Makefile) with the most useful commands to install, test, lint, format and release your project.
-- 📃 Documentation structure using [mkdocs](http://www.mkdocs.org)
-- 💬 Auto generation of change log using **gitchangelog** to keep a HISTORY.md file automatically based on your commit history on every release.
-- 🐋 A simple [Containerfile](Containerfile) to build a container image for your project.  
-  `Containerfile` is a more open standard for building container images than Dockerfile, you can use buildah or docker with this file.
-- 🧪 Testing structure using [pytest](https://docs.pytest.org/en/latest/)
-- ✅ Code linting using [flake8](https://flake8.pycqa.org/en/latest/)
-- 📊 Code coverage reports using [codecov](https://about.codecov.io/sign-up/)
-- 🛳️ Automatic release to [PyPI](https://pypi.org) using [twine](https://twine.readthedocs.io/en/latest/) and github actions.
-- 🎯 Entry points to execute your program using `python -m <wikipedia_name_query>` or `$ wikipedia_name_query` with basic CLI argument parsing.
-- 🔄 Continuous integration using [Github Actions](.github/workflows/) with jobs to lint, test and release your project on Linux, Mac and Windows environments.
+retrieves the age of the specified person
+```
+python Commands.py age --Name "Insert Name"
+```  
+Retreives the full name of a specified person
 
-> Curious about architectural decisions on this template? read [ABOUT_THIS_TEMPLATE.md](ABOUT_THIS_TEMPLATE.md)  
-> If you want to contribute to this template please open an [issue](https://github.com/rochacbruno/python-project-template/issues) or fork and send a PULL REQUEST.
+```
+python Commands.py name --Name "Insert Name"
+```
 
-[❤️ Sponsor this project](https://github.com/sponsors/rochacbruno/)
+retrieves the Date of birth of a specified person
 
-<!--  DELETE THE LINES ABOVE THIS AND WRITE YOUR PROJECT README BELOW -->
+```
+python Commands.py DOB --Name "Insert Name"
+```
+retrieves the date of death of a specified person
 
----
-# wikipedia_name_query
+```
+python Commands.py DOD --Name "Insert Name"
+```
 
-[![codecov](https://codecov.io/gh/M-A-T-T-Y-D/Wikipedia-Name-Query/branch/main/graph/badge.svg?token=Wikipedia-Name-Query_token_here)](https://codecov.io/gh/M-A-T-T-Y-D/Wikipedia-Name-Query)
-[![CI](https://github.com/M-A-T-T-Y-D/Wikipedia-Name-Query/actions/workflows/main.yml/badge.svg)](https://github.com/M-A-T-T-Y-D/Wikipedia-Name-Query/actions/workflows/main.yml)
+outputs everything collected about that person
+```
+python Commands.py Output --Name "Insert Name"
+``` 
 
-Awesome wikipedia_name_query created by M-A-T-T-Y-D
+## Examples of use
+```
+python Commands.py Output --Name "Donald Trump" 
+```
+would return... 
 
-## Install it from PyPI
+Donald Trump 
+Donald Trump Is 77 Years Old  
+Donald Trump was born on: 1946-06-14  
+Donald Trump Still alive  
```

Comment:
Add some triple ticks around this to highlight that it is output to the terminal